### PR TITLE
fetch releases

### PR DESCRIPTION
**Pull request by finestructure**
Ground work for #421, #414

⚠️ Schema change ⚠️

### CODE REVIEW

**Comment by finestructure** [comment]
Good to go now!

**Comment by finestructure** [comment]
Good to go now!

File: Sources/App/configure.swift [hunk 1]
```diff
@@ -104,6 +104,10 @@ public func configure(_ app: Application) throws {
     do {  // Migration 022 - add is_archived to repositories
         app.migrations.add(UpdateRepositoryAddIsArchived())
     }
+    do {
```
Comment (Sherlouk) [inline Sources/App/configure.swift:107]: nit: do we want to continue the pattern of the migration number and comment here?

**Review by daveverwer — COMMENTED** [review]
So I see that we're now storing a publishedAt date which is the release date rather than the date of the commit that the release is made from, that's good. I don't see anything that acts on that data yet though (for the Twitter/RSS feeds, for example), is that correct?

I don't think we should act on that data yet, we need to think about how best to approach the problem, I just want to make sure I understand the change.

**Review by daveverwer — APPROVED** [review]

**Comment by finestructure** [comment]
Yes, that's correct. Just wiring it up and bringing in the schema change while I was touching that entity anyway.

**Pull request merged**